independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/gradle-wrapper/codestart.yml

```diff
@@ -0,0 +1,12 @@
+name: tooling-gradle-wrapper
+type: tooling
+output-strategy:
+  "gradlew": "executable"
+language:
+  base:
+    data:
+      gradle:
+        version: 8.14
```

Comment:
Can we use `{gradle-wrapper.version}` here instead of having a fixed version?

Reply:
`gradle-wrapper.version` has a different version - 9.0.0, currently. The idea is to freeze this version until we remove support for 3.25 and earlier in the tooling.